recipes/methodtools/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "methodtools" %}
+{% set version = "0.4.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9370156e9036789e98cf0e97355b3be3bcd7cc9e520d1e15893a1407719effb2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+  run:
```

Comment:
For python 2, functools32 is required: https://github.com/youknowone/methodtools#installation

As conda-forge doesn't support Python 2 anymore, I would change the bound to 3.6.